File: Service/Implement/CompanyInfoFieldsService.cs
namespace Service.Implement
{
	public class CompanyInfoFieldsService : BaseService<CompanyInfoFields, ICompanyInfoFieldsRepository>
	, ICompanyInfoFieldsService
	{
		private readonly ICompanyInfoFieldsRepository _CompanyInfoFieldsRepository;
		public CompanyInfoFieldsService(ICompanyInfoFieldsRepository CompanyInfoFieldsRepository) : base(CompanyInfoFieldsRepository)
		{
			_CompanyInfoFieldsRepository = CompanyInfoFieldsRepository;
		}

	}
}

File: CMS/src/app/shared/CompanyInfoFields.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { CompanyInfoFields } from 'src/app/shared/CompanyInfoFields.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class CompanyInfoFieldsService extends BaseService{  

    List: CompanyInfoFields[] | undefined;
    ListFilter: CompanyInfoFields[] | undefined;
    FormData!: CompanyInfoFields;    

    DisplayColumns001: string[] = ['STT', 'ID', 'CompanyFieldID', 'Save'];

    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "CompanyInfoFields";
    }
}

